src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -1085,13 +1085,13 @@ ExpressionPtr Visitor::visitTypesafe(Parser::PrimaryExpressionContext* ctx) {
 
   if (ctx->booleanLiteral()) {
     auto b = visitTypesafe(ctx->booleanLiteral());
-    return std::make_unique<sparqlExpression::BoolExpression>(b);
+    return make_unique<BoolExpression>(b);
   }
 
   if (ctx->var()) {
     sparqlExpression::Variable v;
     v._variable = ctx->var()->getText();
-    return std::make_unique<sparqlExpression::VariableExpression>(v);
+    return make_unique<VariableExpression>(v);
```

Comment:
directly initialize the `Variable v {ctx->var()-....`
and then move it in the return, it contains a string.